modules/apps/commerce/commerce-api/src/main/java/com/liferay/commerce/constants/CommercePriceConstants.java

```diff
@@ -0,0 +1,26 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.commerce.constants;
+
+/**
+ * @author Christian Chiappa
+ */
+public class CommercePriceConstants {
+
+	public static final double COMMERCE_MAX_PRICE_VALUE = 999999999.99;
+
+	public static final int COMMERCE_MIN_PRICE_VALUE = 0;
```

Comment:
Why are we using different types here? What if we want the minimum price to be 15.50?